CMakeLists.txt

```diff
@@ -924,6 +924,12 @@ if (NOT DISABLE_SPICY)
         # disable it.
         set(HILTI_DEV_PRECOMPILE_HEADERS OFF)
 
+        # Remove in v6.2: Older versions of Spicy use find_package(Python3).
+        #                 Accommodate by setting the Python3_EXECUTABLE hint.
```

Comment:
Fancy indention 👯‍♂️. If we want it this way, make it flush right, else just default to flush left like elsewhere.